internal/kubernetes/operator/project/project.go

```diff
@@ -704,7 +704,7 @@ func buildTeams(teamsProvider store.AtlasOperatorTeamsStore, orgID, projectID, p
 				teamRef.TeamID, projectName, projectID, err)
 		}
 
-		crName := fmt.Sprintf("%s-team-%s", strings.ToLower(projectName), strings.ToLower(team.Name))
+		crName := fmt.Sprintf("%s-team-%s", resources.NormalizeAtlasResourceName(projectName), resources.NormalizeAtlasResourceName(team.Name))
```

Comment:
you apply the same transformation to `projectName` multiple times, consider centralizing it and reusing the trnasformed value

Reply:
Done

Reply:
Done